css_stylesheet.css

```diff
@@ -56,7 +53,8 @@ header {
 	font-family: "Courier New", serif;
 	color: #303030;
 	float: right;
-	margin-top: 55px;
+	margin-top: 65px;
+	font-size: 1.9em;;
```

Comment:
You're using a mixture of pixels and ems. This might result in things being out of proportion if the user changes the font size. Consider using only one - pixels if you want the website's formatting to remain consistent, ems if you want the website to resize when the user changes the font size.

Reply:
changed. Thanks.